ui-v2/tests/acceptance/components/catalog-filter.feature

```diff
@@ -123,31 +123,6 @@ Feature: components / catalog-filter
     | Model   | Page     | Url                       |
```

Comment:
All changes to tests below are removing user paths that no longer exist or tweaking existing paths that are slightly changed. More tests to come here to test new paths/functionality.